main.cpp

```diff
@@ -53,25 +54,25 @@ int main() {
         {"SKNWTS06", "Warts", 4800.00},
         {"SKNSDT07", "Sensitive Skin Dermal Therapy", 10000.00}
     };
-    
+
     vector<int> quantities(packages.size(), 0); // Initialize quantities with 0
-    
+
     while (!loggedIn) {
         string inputUsername, inputPassword;
         cout << "=== Login ===" << endl;
         cout << "Username: ";
         cin >> inputUsername;
         cout << "Password: ";
         cin >> inputPassword;
-        
+
         if (inputUsername == username && inputPassword == password) {
             loggedIn = true;
             break;
         } else {
             cout << "Invalid username or password. Please try again." << endl;
         }
     }
-    
+
     if (loggedIn) {
         bool exitProgram = false;
         while (!exitProgram) {
```

Comment:
1. `if (loggedIn)`:
   This condition checks whether a user is logged in or not. If the user is logged in, the code inside the if statement will be executed. Otherwise, it will be skipped.

2. `bool exitProgram = false;`:
   This line declares and initializes a boolean variable named `exitProgram` with the value `false`. This variable will be used to control the main program loop.

3. `while (!exitProgram)`:
   This is the start of a while loop that continues executing as long as the `exitProgram` variable is `false`. In other words, the loop will continue until `exitProgram` becomes `true`.

4. Inside the while loop, there is a series of statements that display a menu of options to the user and prompt them to select an option.

5. `cin >> option;`:
   This line reads the user's input from the standard input and stores it in the variable `option`. The user is expected to enter a numerical value corresponding to one of the menu options.

6. `switch (option)`:
   This switch statement is used to perform different actions based on the value of the `option` variable.

7. `case 1:`:
   If the value of `option` is `1`, the code inside this case will be executed. It calculates the total bill based on some packages and quantities, and then saves a receipt to a file named "receipt.txt". The receipt includes information about each package, its quantity, subtotal, and the total amount due.